doc/api-coverage.rst

```diff
@@ -150,6 +150,22 @@ API Coverage
 +------------------------------------------------+---------------------+--------------------+------------+
 | `pyspark.sql.group`_                           | ✘                   | ✔                  |            |
 +------------------------------------------------+---------------------+--------------------+------------+
+| `pyspark.sql.pandas`_                          | ✔                   | ✘                  |            |
```

Comment:
I intended `pandas` to be `Internal` too; however, I am maybe ignorant about the meaning of `Internal` and `Mixed`.